### PR TITLE
configure write/read consistency for cassandra replicator

### PR DESCRIPTION
**Pull request by hulevskyi**

### CODE REVIEW

File: eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/CassandraHealthCheck.scala [hunk 1]
```diff
@@ -19,7 +20,7 @@ object CassandraHealthCheck {
 
   def of[F[_] : Concurrent : Timer : LogOf](
     session: Resource[F, CassandraSession[F]]
-  ): Resource[F, CassandraHealthCheck[F]] = {
+  )(implicit c: ConsistencyConfig.Read): Resource[F, CassandraHealthCheck[F]] = {
```
Comment (t3hnar) [inline eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/CassandraHealthCheck.scala:23]: I really don't like that simplification :)
I'd suggest passing that as `consistencyConfig: ConsistencyConfig.Read` everywhere

File: eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/EventualCassandra.scala [hunk 2]
```diff
@@ -52,8 +55,8 @@ object EventualCassandra {
   ](
     schemaConfig: SchemaConfig,
     origin: Option[Origin],
-    metrics: Option[EventualJournal.Metrics[F]]
-  ): F[EventualJournal[F]] = {
+    metrics: Option[EventualJournal.Metrics[F]],
+  )(implicit c: ConsistencyConfig.Read, w: ConsistencyConfig.Write): F[EventualJournal[F]] = {
```
Comment (t3hnar) [inline eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/EventualCassandra.scala:59]: here I propose to pass single `consistencyConfig: ConsistencyConfig` covering both cases

**Review by t3hnar — CHANGES_REQUESTED** [review]

File: eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/EventualCassandra.scala (hunk 2 above)
Comment (hulevskyi) [reply eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/EventualCassandra.scala]: addressed

File: eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/CassandraHealthCheck.scala (hunk 1 above)
Comment (hulevskyi) [reply eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/CassandraHealthCheck.scala]: addressed

File: eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/EventualCassandra.scala [hunk 3]
```diff
@@ -174,19 +178,19 @@ object EventualCassandra {
 
   object MetaJournalStatements {
 
-    def of[F[_]: Concurrent: Parallel: CassandraSession](
+    def of[F[_]: Concurrent: CassandraSession](
       schema: Schema,
       segmentNrsOf: SegmentNrsOf[F],
       segments: Segments
-    ): F[MetaJournalStatements[F]] = {
+    )(implicit r: ConsistencyConfig.Read): F[MetaJournalStatements[F]] = {
```
Comment (t3hnar) [inline eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/EventualCassandra.scala:185]: it still implicit and still shortnamed, could you please address this as well?

**Review by t3hnar — CHANGES_REQUESTED** [review]

**Pull request merged**